conf/nginx.conf

```
@@ -5,4 +5,19 @@ location __PATH__/ {
     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
     proxy_set_header X-Forwarded-Proto $scheme;
     proxy_pass http://unix:__INSTALL_DIR__/sock;
+
+    if ($request_method = 'OPTIONS') {
+        more_set_headers 'Access-Control-Allow-Origin: *';
```

```suggestion
        more_set_headers 'Access-Control-Allow-Origin: https://__DOMAIN__';
```

Let's go with this for now @_@